server/data/it/frasi.txt

```diff
@@ -11169,4 +11171,5 @@ Le estensioni elencate indicano il formato di default in cui le suite di produtt
 Il formato di MS Office è un formato chiuso e proprietario di Microsoft.
 Lo evolve attraverso le varie versioni di Office in modo che supporti le nuove funzionalità offerte.
 È possibile che documenti creati con versioni precedenti a quella con la quale vengono letti presentino degli errori.
-Free non come "free beer".
\ No newline at end of file
+Free non come "free beer".
+Dare il 100%.
```

Comment:
`%` ->`percento`

Reply:
I was thinking here to leave as reference for the % symbol